src/tests/functional/shared_test_classes/src/single_op/inverse.cpp

```diff
@@ -60,43 +56,76 @@ void InverseLayerTest::SetUp() {
     function = std::make_shared<ov::Model>(results, ParameterVector{data}, "InverseTestCPU");
 }
 
-template <typename T>
-void fill_data(T* dst, const size_t count, const unsigned int seed) {
-    std::mt19937 gen(seed);
-    std::uniform_real_distribution<float> dis(0.0f, 10.0f);
-
-    for (size_t i = 0; i < count; i++) {
-        dst[i] = static_cast<T>(dis(gen));
-    }
-}
-
 void InverseLayerTest::generate_inputs(const std::vector<ov::Shape>& targetInputStaticShapes) {
     inputs.clear();
     const auto& func_inputs = function->inputs();
     const auto& func_input = func_inputs[0];
     const auto& name = func_input.get_node()->get_friendly_name();
     const auto& in_prc = func_input.get_element_type();
-    auto tensor = ov::Tensor(in_prc, targetInputStaticShapes[0]);
-    size_t count = std::accumulate(targetInputStaticShapes[0].begin(),
-                                   targetInputStaticShapes[0].end(),
-                                   1,
-                                   std::multiplies<size_t>());
-
-    switch (in_prc) {
-    case ov::element::f32:
-        fill_data(tensor.data<ov::element_type_traits<ov::element::f32>::value_type>(), count, m_seed);
-        break;
-    case ov::element::f16:
-        fill_data(tensor.data<ov::element_type_traits<ov::element::f16>::value_type>(), count, m_seed);
-        break;
-    case ov::element::bf16:
-        fill_data(tensor.data<ov::element_type_traits<ov::element::bf16>::value_type>(), count, m_seed);
-        break;
-    default:
-        OPENVINO_THROW("Inverse does not support precision ", in_prc, " for the 'data' input.");
-    }
 
+    // ov::test::utils::InputGenerateData in_data;
+    // in_data.start_from = -5.0;
+    // in_data.range = 10;
+    // in_data.resolution = 0;
+    // in_data.seed = m_seed;
+    // auto tensor = ov::test::utils::create_and_fill_tensor(in_prc, targetInputStaticShapes[0], in_data);
```

Comment:
Should it be removed?

Reply:
Reverted back to using the normal fill instead of the consistently version